nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1739,53 +1632,36 @@ define(
             }
 
             function doDeleteCell() {
-                const content = div([
-                    p([
-                        'Deleting this cell will not remove any output cells or data objects it may have created. ',
-                        'Any input parameters or other configuration of this cell will be lost.',
-                    ]),
-                    p(
-                        'Deleting this cell will also cancel any pending jobs, but will leave generated output intact'
-                    ),
-                    blockquote([
-                        'Note: It is not possible to "undo" the deletion of a cell, ',
-                        'but if the Narrative has not been saved you can refresh the browser window ',
-                        'to load the Narrative from its previous state.',
-                    ]),
-                    p('Continue to delete this app cell?'),
-                ]);
-                ui.showConfirmDialog({ title: 'Confirm Cell Deletion', body: content }).then(
-                    (confirmed) => {
-                        if (!confirmed) {
-                            return;
-                        }
+                DialogMessages.showDialog({ action: 'deleteCell' }).then((confirmed) => {
+                    if (!confirmed) {
+                        return;
+                    }
 
-                        const jobState = model.getItem('exec.jobState');
-                        if (jobState) {
-                            cancelJob(jobState.job_id);
-                        }
+                    const jobState = model.getItem('exec.jobState');
+                    if (jobState) {
+                        cancelJob(jobState.job_id);
+                    }
 
-                        // tear down all the sub widgets.
-                        // TODO: make all widget behavior consistent. Either message or promise.
-                        Object.keys(widgets).forEach((widgetId) => {
-                            try {
-                                const widget = widgets[widgetId];
-                                if (widget.stop) {
-                                    widget.stop();
-                                } else {
-                                    widget.instance.bus().send('stop');
-                                }
-                            } catch (ex) {
-                                console.error('ERROR stopping widget', widgetId, ex);
+                    // tear down all the sub widgets.
+                    // TODO: make all widget behavior consistent. Either message or promise.
+                    Object.keys(widgets).forEach((widgetId) => {
```

Comment:
Fairly sure this code isn't used -- I was going to check but forgot about it with all the other deleting I was doing!